sale_order_lot_selection/__openerp__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#########################################################################
+#                                                                       #
+# Copyright (C) 2015  Agile Business Group                              #
+#                                                                       #
+# This program is free software: you can redistribute it and/or modify  #
+# it under the terms of the GNU Affero General Public License as        #
+# published by the Free Software Foundation, either version 3 of the    #
+# License, or (at your option) any later version.                       #
+#                                                                       #
+# This program is distributed in the hope that it will be useful,       #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+# GNU Affero General Public Licensefor more details.                    #
+#                                                                       #
+# You should have received a copy of the                                #
+# GNU Affero General Public License                                     #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. #
+#                                                                       #
+#########################################################################
+
+
+{
+    'name': 'Sale Order Lot Selection',
+    'version': '1.0',
```

Comment:
Please check the guidelines on how to define the module version number.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers